hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/FeeCalculatorImpl.java

```diff
@@ -135,6 +135,13 @@ public FeeCalculator addNetworkRamByteSeconds(long amount) {
         return this;
     }
 
+    @NonNull
+    @Override
+    public FeeCalculator addStorageBytesHours(long seconds) {
```

Comment:
If this is always seconds, and the underlying call is seconds, why is it named `addStorageBytesHours`?

Reply:
I think it comes from some abbreviation used in the mono code. I can change it to secs.